lib/DistributionProvider.pm

```diff
@@ -20,6 +20,7 @@ use Distribution::Sle::12;
 use Distribution::Opensuse::Leap::42;
 use Distribution::Opensuse::Leap::15;
 use Distribution::Opensuse::Tumbleweed;
+use Distribution::Alp::Alp;
```

Comment:
just one Alp, we don't need a folder if there is only one file and we don't know exactly the flavors of ALP at this point.